src/screens/PlrStaking/RouteCard.tsx

```diff
@@ -0,0 +1,284 @@
+
+// Hooks
+import { useTransactionsEstimate } from 'hooks/transactions';
+
+// Components
+import TokenIcon from 'components/display/TokenIcon';
+import RadioButton from 'components/RadioButton';
+import Text from 'components/core/Text';
+import { BigNumber, ethers } from 'ethers';
+
+interface IRouteCard {
+  offer?: any;
+  selected?: boolean;
+  chain?: Chain;
+  plrToken?: AssetOption;
+  formattedToAmount?: string;
+  formattedFromAmount?: string;
+  networkName?: string;
+  provider?: any;
+  highFee?: boolean;
+  onSelectOffer?: (offer: any, feeInfo: TransactionFeeInfo | null) => void;
+  disabled?: boolean;
+  stakeFeeInfo: any;
+  stakeGasFeeAsset: Asset | AssetOption;
+  transactions: any;
+  gasFeeAsset: Asset | AssetOption;
+}
+
+const RouteCard: FC<IRouteCard> = ({
+  offer,
+  selected,
+  chain,
+  plrToken,
+  formattedToAmount,
+  formattedFromAmount,
+  networkName,
+  provider,
+  onSelectOffer,
+  disabled,
+  stakeFeeInfo,
+  stakeGasFeeAsset,
+  transactions,
+  gasFeeAsset,
+}) => {
+  const currency = useFiatCurrency();
+  const chainRates = useChainRates(chain);
+  const ethRates = useChainRates(CHAIN.ETHEREUM);
+
+  const config = useProviderConfig(provider);
+
+  const { feeInfo } = useTransactionsEstimate(chain, transactions, true, gasFeeAsset);
+
+  const feeEtherValueBn = feeInfo?.fee ? BigNumber.from(feeInfo.fee.toString()) : null;
+  const stakeFeeEtherValueBn = stakeFeeInfo ? BigNumber.from(stakeFeeInfo.toString()) : null;
+
+  const getFiatValue = (value: BigNumber, address: string, isEthereum?: boolean) => {
+    if (!value) return null;
+
+    const etherValue = ethers.utils.formatEther(value);
+    const valueInFiat = getBalanceInFiat(currency, etherValue, !!isEthereum ? ethRates : chainRates, address);
+    return valueInFiat;
+  };
+
+  const getTotalGasFees = () => {
+    let totalFiatValue = 0;
+    if (feeEtherValueBn) totalFiatValue += getFiatValue(feeEtherValueBn, gasFeeAsset.address) || 0;
+    if (stakeFeeEtherValueBn) totalFiatValue += getFiatValue(stakeFeeEtherValueBn, stakeGasFeeAsset.address, true) || 0;
+    return totalFiatValue;
+  };
+
+  return (
+    <RouteWrapper>
+      <RouteContainer onPress={() => onSelectOffer?.(offer, feeInfo)} disabled={disabled || !onSelectOffer}>
+        <IconWrapper>{plrToken && <TokenIcon url={plrToken?.iconUrl} size={48} chain={plrToken?.chain} />}</IconWrapper>
+
+        <RouteInfoContainer>
+          <RouteInfoRow>
+            <MainText>{formattedToAmount}</MainText>
+            <MainText highlighted>{`on ${networkName}`}</MainText>
+          </RouteInfoRow>
+
+          <RouteInfoRow>
+            <GasPriceWrapper>
+              <SubText>
+                <HighlightText>{'Est. fee: '}</HighlightText>
+                {formatFiatValue(getTotalGasFees(), currency)}
+              </SubText>
+            </GasPriceWrapper>
+            <SubText>
+              <HighlightText>Est. time:</HighlightText>
+              {` 2 mins`}
```

Comment:
Is this supposed to be dynamic in any way?

Reply:
Honestly not sure, I couldn't seen any time estimates on the object from the bridge - but I don't know if there's a different way to pull these down

Reply:
Let's comment this out for now with a note, and we'll make a decision whether to add this back in